mne/beamformer/_lcmv.py

```diff
@@ -721,7 +721,8 @@ def tf_lcmv(epochs, forward, noise_covs, tmin, tmax, tstep, win_lengths,
     raw_picks = [raw.ch_names.index(c) for c in ch_names]
 
     # Make sure epochs.events contains only good events:
-    epochs.drop_bad_epochs()
+    if epochs._bad_dropped is False:
+        epochs.drop_bad_epochs()
```

Comment:
hum... it means you changed the semantics...

if I call twice drop_bad_epochs nothing should happen the second time.